EPFL_custom_editor_menu_loader.php

```diff
@@ -10,3 +10,14 @@
 
 
 require (WPMU_PLUGIN_DIR.'/epfl-custom-editor-menu/epfl-custom-editor-menu.php');
+
+
+function remove_redirection_from_tools_menu_for_editor() {
+    $user = wp_get_current_user();
+    if (!in_array('administrator', $user->roles)) {
+	    remove_submenu_page('tools.php', 'redirection.php');
```

Comment:
@dragonleman please keep the same indentation.

Reply:
Done in [f8a911b8681d6bf0d7b39a667f255f9ae2cf865e](https://github.com/epfl-si/wp-mu-plugins/commit/f8a911b8681d6bf0d7b39a667f255f9ae2cf865e)